intezeranalyze_connector.py

```diff
@@ -124,6 +128,19 @@ def _make_rest_call(self, endpoint, action_result, headers=None, params=None, da
         # Create a URL to connect to
         url = self._base_url + endpoint
 
+        if endpoint != 'is-available' and not self._jwt_token:
+            response = requests.post(self._base_url + 'v2-0/get-access-token', json={'api_key': config.get('apiKey')})
```

Comment:
i thins we can add the v2-0 to the base url

Reply:
we can't because of the is-available endpoint which is without v2-0